src/fx/Alert.js

```diff
@@ -0,0 +1,21 @@
+function alertEffect(dispatch, message) {
+  window.alert(message)
+}
+
+/**
+ * Describes an effect that will call [`window.alert`](https://developer.mozilla.org/en-US/docs/Web/API/Window/alert) to display a dialog window with the specified message and an "OK" button.
+ *
+ * @memberof module:fx
+ * @param {...*} message - message to display in the alert dialog
```

Comment:
I believe [`alert`](https://developer.mozilla.org/en-US/docs/Web/API/Window/alert) only has a single parameter.

Reply:
Tried resolving this in b6bec4d. Is a type of `{*}` more appropriate?